### ISSUE TITLE
Incorrect plutus tag for Week05.

### ISSUE THREAD

**Issue by jhodgdev** [input-output-hk/plutus-pioneer-program#60]
The provided Plutus git tag in Week05 (`8f1a47674a99ac9bc2aba3231375d8d6de0641d2`) results in the following error:

```
~/repos/plutus(master) » git checkout 8f1a47674a99ac9bc2aba3231375d8d6de0641d2
fatal: reference is not a tree: 8f1a47674a99ac9bc2aba3231375d8d6de0641d2
```

**Comment by jdahodgkinson** [comment]
Fixed by #36.

**Comment by luisantonioig** [comment]
Hi @jhodgdev,

I hope this message finds you well.

I wanted to inform you that we have decided to close this issue as it pertains to an old branch of the course. Please note that these branches are not actively maintained and are kept for historical purposes.

We encourage you to utilize the branch that best suits your needs, whether it's the current version or any other relevant branch.

Please don't hesitate to reach out if you have any questions or require further assistance. We're here to help!

Thank you for your understanding and your valuable contributions.
